legend-sdlc-generation-file/src/test/java/org/finos/legend/sdlc/generation/file/TestFileGenerationFactory.java

```diff
@@ -95,20 +95,26 @@ public void testFileGenerationFactory()
         testAvroOutput(avroResult);
 
         // protobuf
-        List<GenerationOutput> protoBufResult = result.get(specifications.get("generation::MyProtobuf"));
-        Assert.assertEquals(2, protoBufResult.size());
-        MapIterable<String, GenerationOutput> protobufOutputs = LazyIterate.adapt(protoBufResult).groupByUniqueKey(GenerationOutput::getFileName);
-        Assert.assertEquals("message Person {\n" +
-                " string firstname = 1;\n" +
-                " string lastname = 2;\n" +
-                "}", protobufOutputs.get("model/Person.proto").getContent());
-        Assert.assertEquals("message Firm {\n" +
-                " Person employees = 1;\n" +
-                "}\n" +
-                "message Person {\n" +
-                " string firstname = 1;\n" +
-                " string lastname = 2;\n" +
-                "}", protobufOutputs.get("model/Firm.proto").getContent());
+         List<GenerationOutput> protoBufResult = result.get(specifications.get("generation::MyProtobuf"));
+         Assert.assertEquals(2, protoBufResult.size());
+         MapIterable<String, GenerationOutput> protobufOutputs = LazyIterate.adapt(protoBufResult).groupByUniqueKey(GenerationOutput::getFileName);
+         Assert.assertEquals("syntax = \"proto3\";\n" +
+                 "import \"google/protobuf/timestamp.proto\";\n" +
+                 "package model;\n" +
+                 "message Person {\n" +
+                 " string firstname = 1;\n" +
+                 " string lastname = 2;\n" +
+                 "}", protobufOutputs.get("model_Person.proto").getContent());
+         Assert.assertEquals("syntax = \"proto3\";\n" +
+                 "import \"google/protobuf/timestamp.proto\";\n" +
+                 "package model;\n" +
+                 "message Firm {\n" +
+                 " Person employees = 1;\n" +
+                 "}\n" +
+                 "message Person {\n" +
+                 " string firstname = 1;\n" +
+                 " string lastname = 2;\n" +
+                 "}", protobufOutputs.get("model_Firm.proto").getContent());
```

Comment:
The indentation looks off here.

Reply:
I kept the same indentation as before.  Should I just let IntelliJ indent it?

Reply:
It looks to me like you pushed everything in 1 space. Previously, the first non-whitespace character on line 98 was in column 9, but now it's in column 10.